src/kinetics/KineticsFactory.cpp

```diff
@@ -6,7 +6,11 @@
 // at https://cantera.org/license.txt for license and copyright information.
 
 #include "cantera/kinetics/KineticsFactory.h"
-#include "cantera/kinetics/GasKinetics.h"
+#include "cantera/kinetics/BulkKinetics.h"
+
+#define CT_SKIP_DEPRECATION_WARNINGS
```

Comment:
Is there a reason to still include `GasKinetics.h`?

Reply:
The "gentle" deprecation approach that I've tried to implement here leaves setting `kinetics: gas` in an input file creating a `GasKinetics` object in Cantera 3.0, then switching that to `BulkKinetics` in Cantera 3.1. The reasons for this are:
1. I don't want to make users edit their input files -- `gas` can remain a synonym for `bulk` indefinitely
2. If a user explicitly asks for a `gas` object, in Cantera 3.0, they should still get one -- they could (at least in theory) `dynamic_cast` that pointer to `GasKinetics`, and that should only raise a deprecation warning for now.

The way I've achieved this (I think -- please feel free to check this logic) is to have the `gas` model continue to create a `GasKinetics` object without any warning. However, if you `#include "GasKinetics.h`, which is a prerequisite for directly creating or casting to `GasKinetics`, you will get a compiler warning from that input file via the `#pragma message` and runtime warnings from the methods that are only defined for the `GasKinetics` class.

Reply:
Ok, this makes sense. My own, less "gentle", intuition would have been to switch this over directly, as we are bumping the major version. I don't have a problem with the gentle approach though.